test/e2e_node/swap_test.go

```diff
@@ -56,7 +57,7 @@ var (
 	noLimits *resource.Quantity = nil
 )
 
-var _ = SIGDescribe("Swap", "[LinuxOnly]", feature.Swap, framework.WithSerial(), func() {
+var _ = SIGDescribe("Swap", "[LinuxOnly]", ginkgo.Ordered, feature.Swap, framework.WithSerial(), func() {
```

Comment:
What is ginko ordered?

Reply:
This will run the tests in the order they appear in the file. These tests are very delicate, and I found this to be of dramatic stabilization while working on https://github.com/kubernetes/kubernetes/pull/129578.